vignettes/rmarkdown.Rmd

```diff
@@ -120,10 +120,10 @@ rprintln!("x = {}, y = {}", x, y);
 
 ## Exporting Rust functions to R
```

Comment:
Should this also be changed to "functions and objects"?

Reply:
Yes, probably, thanks!